### DISCUSSION TITLE
Is this a substitute of mLauncher?

### DISCUSSION THREAD

**Discussion by i-penr** [DroidWorksStudio/EasyLauncher#112] [Q&A]
Hey :)

I have just noticed this project, and I was wondering about its purpose. Is it supposed to be an updated fleshed-out version of mLauncher, or is it its own separate thing? I was wondering this because they look very similar to each other (as of right now at least).

**Comment by CreativeCodeCat** [comment]
It's more up to date without being a fork of olauncher but yes it is very very similar except imho but just a bit cleaner.
